### ISSUE TITLE
VAE loss

### ISSUE THREAD

**Issue by hamid54** [autonomousvision/sledge#4]
Hi,
I think there is a mistake in _sledge/autoencoder/modeling/objectives/vae_objective.py:55_ . The arguments of binary_cross_entropy_with_logits may need to be swapped. 

Thank you for sharing your interesting work!

Hamid

**Comment by DanielDauner** [comment]
Hi @hamid54, you're right. We accidentally introduced the bug during refactoring. We will fix the ordering in the upcoming release. Thanks for pointing it out!
